### PR TITLE
Add some phpdoc about exceptions

### PR DESCRIPTION
**Pull request by VincentLanglet**
When using static analysis with `AdminInterface`, I get error when I want to `try/catch` these methods, since, according to the phpdoc, they are not throwing any errors.

### CODE REVIEW

File: src/Admin/AdminInterface.php [hunk 1]
```diff
@@ -148,7 +148,7 @@ public function setParentFieldDescription(FieldDescriptionInterface $parentField
     /**
      * Get parent field description.
      *
-     * @return FieldDescriptionInterface The parent field description
+     * @return FieldDescriptionInterface|null The parent field description
```
Comment (core23) [inline src/Admin/AdminInterface.php:151]: IMHO you can remove the comment as it is self-explaining

**Review by core23 — CHANGES_REQUESTED** [review]

File: src/Admin/AdminInterface.php (hunk 1 above)
Comment (VincentLanglet) [reply src/Admin/AdminInterface.php]: Done

**Review by phansys — APPROVED** [review]

**Review by jordisala1991 — APPROVED** [review]

**Review by franmomu — APPROVED** [review]

**Comment by phansys** [comment]
Couldn't these changes target `3.x`?

**Comment by VincentLanglet** [comment]
> Couldn't these changes target `3.x`?

These methods does not exists in 3.x

**Review by OskarStark — APPROVED** [review]

**Review by core23 — APPROVED** [review]

**Comment by phansys** [comment]
Thank you @VincentLanglet!

**Pull request merged**